packages/wonder-blocks-banner/src/components/__tests__/banner.test.js

```diff
@@ -0,0 +1,162 @@
+                text=""
+                actions={[{title: "some link", href: "/", onClick: () => {}}]}
+            />,
+        );
+
+        // Assert
+        const link = screen.getByRole("link");
+        expect(link).toBeInTheDocument();
+    });
+
+    test("passing an onClick without href into an `actions` element causes a Button to appear", () => {
+        // Arrange
+
+        // Act
+        render(
+            <Banner
+                text=""
+                actions={[{title: "some button", onClick: () => {}}]}
+            />,
+        );
+
+        // Assert
+        const link = screen.getByRole("button");
+        expect(link).toBeInTheDocument();
+    });
+
+    test("passing multiple actions causes multiple actions to appear", async () => {
+        // Arrange
+
+        // Act
+        render(
+            <Banner
+                text=""
+                actions={[
+                    {title: "button 1", onClick: () => {}},
+                    {title: "button 2", onClick: () => {}},
+                    {title: "button 3", onClick: () => {}},
+                ]}
+            />,
+        );
+
+        // Assert
+        const buttons = await screen.findAllByRole("button");
+        expect(buttons).toHaveLength(3);
+    });
+
+    test("clicking a button triggers the onClick from the action prop", () => {
+        // Arrange
+        const actionSpy = jest.fn();
+
+        render(
+            <Banner
+                text=""
+                actions={[{title: "a button", onClick: actionSpy}]}
+            />,
+        );
+
+        // Act
+        const button = screen.getByRole("button");
+        button.click();
+
+        // Assert
+        expect(actionSpy).toHaveBeenCalled();
+    });
+
+    test("clicking a link triggers the onClick from the action prop", () => {
+        // Arrange
+        const actionSpy = jest.fn();
+
+        render(
+            <Banner
+                text=""
+                actions={[{title: "a link", onClick: actionSpy, href: "/"}]}
+            />,
+        );
+
+        // Act
+        const link = screen.getByRole("link");
+        link.click();
+
+        // Assert
+        expect(actionSpy).toHaveBeenCalled();
+    });
+
+    test("action href becomes link href", () => {
+        // Arrange
+
+        render(
+            <Banner
+                text=""
+                actions={[{title: "a button", onClick: () => {}, href: "/foo"}]}
+            />,
+        );
+
+        // Act
+        const link = screen.getByRole("link");
+
+        // Assert
+        expect(link).toHaveAttribute("href", "/foo");
+    });
+
+    test("passing an action with no href or onClick throws an error", () => {
+        // Arrange
+
+        // Act
+        const actionTest = () =>
+            render(<Banner text="" actions={[{title: "some button"}]} />);
+
+        // Assert
+        expect(actionTest).toThrow(
+            "An action must have an href or an onClick field.",
+        );
+    });
```

Comment:
**thought:** I'm seeing that we are not including tests for `kind`, but maybe you are planning to add them in the next PR?